### PR TITLE
[Docs] Improve description of set_tracking 

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/monitoring/initial-setup-configuration.ipynb [hunk 1]
```diff
@@ -28,8 +28,10 @@
...
     "\n",
     "Model activities can be tracked into a real-time stream and time-series DB. The monitoring data\n",
-    "is used to create real-time dashboards and track model accuracy and drift. \n",
-    "To set the tracking stream options, specify the following function spec attributes:\n",
+    "is used to create real-time dashboards, track model accuracy, detect drift, and analyze performance. \n",
+    "\n",
+    "To monitor a deployed model, apply `set_tracking()` on your serving function and specify the function spec attributes:\n",
+    "\n",
     "    \n",
     "   `fn.set_tracking(stream_path, batch, sample)`\n",
```
Comment (Eyal-Danieli) [inline docs/monitoring/initial-setup-configuration.ipynb:36]: `tracking_policy` param is missing from the function signature 
```suggestion
    "   `fn.set_tracking(stream_path, batch, sample, tracking_policy)`\n",
```

Comment (Eyal-Danieli) [reply docs/monitoring/initial-setup-configuration.ipynb:36]: In addition, you can add a description for the `tracking_policy` param in the next lines (can't comment on unchanged lines in PR review):
**`tracking_policy`** - optional, model tracking configurations, such as setting the scheduling policy of the model monitoring batch job.

File: docs/serving/custom-model-serving-class.md [hunk 2]
```diff
@@ -167,8 +167,9 @@ You can also deploy a model from within an ML pipeline (check the various demos
 ## Model monitoring
 
 Model activities can be tracked into a real-time stream and time-series DB. The monitoring data
-is used to create real-time dashboards and track model accuracy and drift. 
-To set the tracking stream options, specify the following function spec attributes:
+is used to create real-time dashboards, track model accuracy, detect drift, and analyze performance.
```
Comment (Eyal-Danieli) [inline docs/serving/custom-model-serving-class.md:170]:
```suggestion
is used to create real-time dashboards, detect drift, and analyze performance. 
```

File: docs/serving/custom-model-serving-class.md [hunk 3]
```diff
@@ -167,8 +167,9 @@ You can also deploy a model from within an ML pipeline (check the various demos
 ## Model monitoring
 
 Model activities can be tracked into a real-time stream and time-series DB. The monitoring data
-is used to create real-time dashboards and track model accuracy and drift. 
-To set the tracking stream options, specify the following function spec attributes:
+is used to create real-time dashboards, track model accuracy, detect drift, and analyze performance. 
+
+To monitor a deployed model, apply `set_tracking()` on your serving function and specify the function spec attributes:
 
         fn.set_tracking(stream_path, batch, sample)
```
Comment (Eyal-Danieli) [inline docs/serving/custom-model-serving-class.md:174]: same

```suggestion
        fn.set_tracking(stream_path, batch, sample, tracking_policy)
```

+ add the following description:
`tracking_policy` - optional, model tracking configurations, such as setting the scheduling policy of the model monitoring batch job.

**Review by Eyal-Danieli — CHANGES_REQUESTED** [review]
few little comments, mainly related to a missing param called `tracking_policy`

File: docs/serving/custom-model-serving-class.md (hunk 3 above)
Comment (jillnogold) [reply docs/serving/custom-model-serving-class.md:174]: added description

**Review by Eyal-Danieli — APPROVED** [review]

**Review by assaf758 — APPROVED** [review]

**Pull request merged**